doc/config.rst

```diff
@@ -295,6 +295,21 @@ General configuration
 
    .. versionadded:: 1.3
 
+.. confval:: manpages_url_pattern
+
+   A URL to cross-reference :rst:role:`manpage` directives. If this is
+   defined to ``https://manpages.debian.org/{full}``, the
+   ``:manpage:man(1)`` role will like to
```

Comment:
Please use
```
:literal:`:manpage:`man(1)``
```
instead. It allows to represent role notations correctly.

Reply:
that's quite a mouthful! but okay, done. :)